enclave-tls/VERSION

```diff
@@ -1 +1 @@
-0.1.0
\ No newline at end of file
+0.6.4
```

Comment:
Pre my understanding, this should be modular version of enclave-tls and it may not align with the release version of IC. 
And i saw it is 0.6.3 in epm and here is 0.6.4. 
Suggest to unify them as the same modular version value.